### PR TITLE
Jace Boss dungeon

### PR DESCRIPTION
**Pull request by Simisays**
It seems like ONE jumpstart isn't working on Adventure again . I'll try to get a fix in next PR

### CODE REVIEW

**Comment by kevlahnota** [comment]
I'm trying to recreate the two card art, already found the images, this looks good.

File: forge-gui/res/adventure/Shandalar/custom_cards/jaces_signature_hoodie.txt [hunk 1]
```diff
@@ -0,0 +1,10 @@
...
+SVar:DBConjure:DB$ MakeCard | Conjure$ True | DefinedName$ ChosenCard | Zone$ Battlefield | RememberMade$ True | SubAbility$ DBClearChosen
+SVar:DBClearChosen:DB$ Cleanup | ClearChosenCard$ True | SubAbility$ DBEffect
+SVar:DBEffect:DB$ Effect | RememberObjects$ Remembered | StaticAbilities$ PerpetualAbility | Duration$ Permanent | Name$ Jace's Signature Hoodie's Perpetual Effect
+SVar:PerpetualAbility:Mode$ Continuous | Affected$ Card.IsRemembered | EffectZone$ Command | AffectedZone$ Battlefield,Hand,Graveyard,Exile,Stack,Library,Command | SetPower$ 2 | SetToughness$ 2 | RemoveCardTypes$ True | SetColor$ Blue | AddType$ Creature & Illusion | Description$ That duplicate perpetually has base power and toughness 2/2 and becomes an Illusion creature. 
+Oracle:Provided by Jace's Signature Hoodie\n{E}{E}, Exile Jace's Signature Hoodie: Conjure a duplicate of a random creature card from your opponent's library unto the battlefield under your control. The duplicate perpetually becomes a blue Illusion creature with base power and toughness 2/2. Activate only once each game.
```
Comment (kevlahnota) [inline forge-gui/res/adventure/Shandalar/custom_cards/jaces_signature_hoodie.txt:10]: shard notation is {M}

Comment (kevlahnota) [inline forge-gui/res/adventure/Shandalar/custom_cards/jaces_signature_hoodie.txt:10]: Activate only once each game. shouldn't be needed I think

**Comment by Agetian** [comment]
:+1:

**Pull request merged**